.circleci/config.yml

```diff
@@ -25,13 +25,21 @@ jobs:
 
   validate_production_schema:
     executor: node/build
+    parameters:
+      practice_web:
+        type: string
+        default: C2TQ4PT8R
```

Comment:
Mind adding a comment here pointing to a link where this channel string was found? Will help future devs if this needs to be moved around.